sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/OneCollectorIngestion.java

```diff
@@ -131,6 +141,9 @@ public ServiceCall sendAsync(String appSecret, UUID installId, LogContainer logC
             apiKey.deleteCharAt(apiKey.length() - 1);
         }
         headers.put(API_KEY, apiKey.toString());
+        if (authToken != null && authToken.length() > 0) {
```

Comment:
We don't need `authToken` for One Collector requests, right? I don't think it should be added here.

Reply:
:+1: